amplify-tools-gradle-plugin/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=amplify-tools-gradle-plugin
+POM_NAME=Amplify Tools for Android - Gradle Plugin
+POM_DESCRIPTION=Gradle plugin for the amplify flow with android
```

Comment:
This doesn't really say anything new. Can you change this to like:
```
A Gradle plugin that enables the user to <achieve outcomes, here>.
```
Also, please capitalize `Amplify` and `Android` which are proper nouns